<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addCityButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add City</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cityLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="endingCityLineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kilometerLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ending City</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance (km)</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadFile">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload File</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cityLineEdit2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="foodLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>310</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="costLineEdit">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>310</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>290</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>290</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Food</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>290</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cost</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFoodButton">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>310</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Food</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadFile2">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>350</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSales_Report">
   <property name="text">
    <string>Sales Report</string>
   </property>
  </action>
  <action name="actionPurchases_Char">
   <property name="text">
    <string>Purchases Char</string>
   </property>
  </action>
  <action name="actionPurchases_Chart">
   <property name="text">
    <string>Purchases Chart</string>
   </property>
  </action>
  <action name="actionMember_Purchases">
   <property name="text">
    <string>Member Purchases</string>
   </property>
  </action>
  <action name="actionManage_Inventory">
   <property name="text">
    <string>Manage Inventory</string>
   </property>
  </action>
  <action name="actionItem_List">
   <property name="text">
    <string>Inventory List</string>
   </property>
  </action>
  <action name="actionItem_Search">
   <property name="text">
    <string>Item Search</string>
   </property>
  </action>
  <action name="actionManage_Members">
   <property name="text">
    <string>Manage Members</string>
   </property>
  </action>
  <action name="actionMember_Search">
   <property name="text">
    <string>Member Search</string>
   </property>
  </action>
  <action name="actionUpgrade_Downgrade_List">
   <property name="text">
    <string>Upgrade/Downgrade List</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
